### DISCUSSION TITLE
Which version of Laravel 8 should I use?

### DISCUSSION THREAD

**Discussion by suyar** [Laravel-Lang/common#30] [Q&A] [answered]
Which version of Laravel 8 should I use?

Is it `~8.0`?

How should I use the latest version, just like 10.1?

![image](https://user-images.githubusercontent.com/15704489/130044373-331dc820-1272-4af3-8068-792b8ab185ee.png)

**Comment by mohamedsabil83** [comment, marked as answer]
Just run `composer require laravel-lang/lang` to install the latest version

**Reply by andrey-helldar** [reply]
@suyar, even better, use the [`official publisher`](https://github.com/Laravel-Lang/publisher).
